app/locations/controllers.js

```diff
@@ -0,0 +1,28 @@
+const { find, groupBy, sortBy } = require('lodash')
+
+function _getUserLocations(user) {
```

Comment:
I don't feel we need to specify functions as private in this way. This isn't a class. 
I'd also agree with Dom I don't feel the `_getUserLocations` is adding to the clarity of the code and this abstraction is only being used in one place and there is a bit of jumping around which is harder to read. Feels like it would be clearer in the `locations` function